packages/teams-js/src/internal/internalAPIs.ts

```diff
@@ -91,13 +91,17 @@ export function isCurrentSDKVersionAtLeast(requiredVersion: string = defaultSDKV
 
 /**
  * @hidden
- * Helper function to identify if host client is either android or ios
+ * Helper function to identify if host client is either android, ios, or ipados
  *
  * @internal
  * Limited to Microsoft-internal use
  */
 export function isHostClientMobile(): boolean {
-  return GlobalVars.hostClientType == HostClientType.android || GlobalVars.hostClientType == HostClientType.ios;
+  return (
+    GlobalVars.hostClientType == HostClientType.android ||
+    GlobalVars.hostClientType == HostClientType.ios ||
+    GlobalVars.hostClientType == HostClientType.ipados
+  );
```

Comment:
The bug https://office.visualstudio.com/MetaOS/_workitems/edit/7715609 mentions teamsphones.
I have checked tat in teams repo but not able to find any entries. I'd image it maybe one of depreciated stuff but will double check with Yuri and update the PR accordingly if needed.

Reply:
This change makes sense, but wanted to check: After this change some functions in media.ts will work differently on ipad. After a quick scan, I think it is mainly media.selectMedia. I *think* that it would have thrown an exception here on ipad before your change:

![image](https://user-images.githubusercontent.com/18446904/228303489-f1511d24-71c5-4292-9a74-e713b4bd7e70.png)

Does this function now work on ipad? Is it supposed to? I *assume* it's fine, but wanted to check and see what you thought

Reply:
well, it looks media is really a complex api(if not messy).It feels it should be fine mainly because of the line from 606-610 I think.

With adding ipados in isHostClient, it will go to else section in this screenshot
<img width="763" alt="image" src="https://user-images.githubusercontent.com/8062622/228334332-2af294e3-b238-4fe6-bdd4-213082650e0e.png">
If it goes to else, it should be caught earlier from line 606 since they are doing the same thing.